### ISSUE TITLE
Releasing shared-subject pre-trained model

### ISSUE THREAD

**Issue by Concarne2** [MedARC-AI/MindEyeV2#22]
Hello,

I was wondering if you guys could release (or point to the location if it is already released) the 4096-dim pre-trained shared-subject model that is not fine-tuned to a single subject? I'm assuming the "multisubject_subj01_1024hid_nolow_300ep" model in Huggingface is the 1024-dim version of the model, and it would be a great help if you guys could also release the 4096-dim version.

Thanks in advance!

**Comment by PaulScotti** [comment]
Sorry for the delay, these checkpoints are all now available on huggingface (see final_multisubject_subj0# folders): https://huggingface.co/datasets/pscotti/mindeyev2/tree/main/train_logs

I also just now updated the main repo with better instructions in the README and cleaned up code throughout the repo as well.

Best,
Paul
